### ISSUE TITLE
Embed Architecture diagram in app

### ISSUE THREAD

**Issue by leonicus** [devopsloft/devopsloft#326]
Now that we have a new Architecture document, it should be embedded in the app to enhance its accessibility
The diagram can be found here 
https://www.lucidchart.com/invitations/accept/c25b7de6-9321-4e16-afb1-2a586e78e188

**Comment by lioramilbaum** [comment]
We should also deprecate the previous architecture document
